src/iframe/preview.js

```diff
@@ -65,12 +72,30 @@ const Share = {
 const State = {
   liveStateNeeded: Boolean(getCookie('is-logged-in')) || Boolean(getCookie('io.prismic.previewSession')),
```

Comment:
Check if this is still necessary (if the toolbar is not loaded when no cookie is found, we can get rid of this @arnaudlewis)